ij/kotlin-psi/src/main/kotlin/org/jetbrains/academy/test/system/kotlin/ij/analyzer/IjCodeAnalyzerUtil.kt

```diff
@@ -1,4 +1,4 @@
-package org.jetbrains.academy.test.system.ij.analyzer
+package org.jetbrains.academy.test.system.kotlin.ij.analyzer
```

Comment:
Why? It works with KtElements

Reply:
Because now you have a lot of the same methods in Java and Kotlin modules, e.g.
```
fun PsiFile.hasClass(className: String): Boolean = hasElementOfTypeWithName(PsiClass::class.java, className
```
and
```
fun PsiFile.hasClass(className: String): Boolean = hasElementOfTypeWithName(KtClass::class.java, className)
```

Instead it you can move these methods in the ij core module and here just call them with necessary arguments